### PR TITLE
refactor: refactor honeybadger component

### PR DESCRIPTION
**Pull request by zeroFruit**
## Description

refactor honeybadger

## Checklist

Thank you for your contribution.
Before submitting this PR, please make sure:

- [x] Applying goimports
- [x] Test case
- [x] End of Work

### CODE REVIEW

File: honeybadger/acs_map.go [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+
+import (
+	"sync"
+
+	"github.com/DE-labtory/cleisthenes"
+)
+
+type ACSMap struct {
+	lock  sync.WaitGroup
+	items map[cleisthenes.Address]cleisthenes.ACS
```
Comment (zeroFruit) [inline honeybadger/acs_map.go]: Address -> epoch

Comment (AgwaB) [inline honeybadger/acs_map.go]: `cleisthenes.Address` -> `epoch`

File: queue.go [hunk 2]
```diff
@@ -86,9 +88,90 @@ func (q *memQueue) at(index int) (Transaction, error) {
...
+	batch, cleanUp := m.selectRandomTx(candidate, int(b/n))
+	defer cleanUp()
+
+	return Batch{txList: batch}, nil
+}
+
+// loadCandidateTx is a function that returns transactions from queue
+func (m *DefaultTxQueueManager) loadCandidateTx(candSize int) ([]Transaction, error) {
+	candidate := make([]Transaction, candSize)
+	var err error
```
Comment (AgwaB) [inline queue.go:142]: what is it for?

File: honeybadger/honeybadger.go [hunk 3]
```diff
@@ -0,0 +1,72 @@
+package honeybadger
+
+import (
+	"sync/atomic"
+
+	"github.com/DE-labtory/cleisthenes"
+	"github.com/DE-labtory/cleisthenes/pb"
+)
+
+type batchTransporter struct {
```
Comment (AgwaB) [inline honeybadger/honeybadger.go]: How about just `batchTransport`?

File: transaction.go [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+	txList []Transaction
+}
+
+// TxList is a function returns the transaction list on batch
+func (batch *Batch) TxList() []Transaction {
+	return batch.txList
+}
+
+type BatchReceiver interface {
+	ReceiveBatch(batch Batch)
```
Comment (AgwaB) [inline transaction.go]: How about `Receive`?

**Review by AgwaB — DISMISSED** [review]

File: honeybadger/honeybadger.go (hunk 3 above)
Comment (zeroFruit) [reply honeybadger/honeybadger.go]: good!

File: transaction.go (hunk 4 above)
Comment (zeroFruit) [reply transaction.go]: fixed!

**Pull request merged**